types/account.d.ts

```diff
@@ -218,15 +220,42 @@ export type UpdateCustomerArgs = {
   fullName?: string;
 };
 
-export type Consent = {
+export interface CleengConsent {
```

Comment:
I think it's better to resolve this on service level with formatter methods as the previous functionalities.. We don't need separate types, since there may be new integrations in the future and it will become harder and messy with checks and conditions.
In short, we should avoid using cleeng or jw.

Reply:
I agree about resolving it on service level, which only means moving this type in the service file where it belongs (in this case `cleeng.account.service.ts`), but I don't think formatter functions are a better solution than keeping a clean type of what a Cleeng consent looks like and keep the logic of how it's formatted unexposed and contained in the only place it's ever to be used - the `getPublisherConsents` function. It's a simple straight-forward logic and it wouldn't be used from anywhere else, because that's the reason we're doing the formatting this early on - so that we avoid possible breakage due to differences between `cleeng` consents and `jwp`/`inplayer` register fields. Once that's in place, extracting that logic into a separate function is not only unnecessary, but can trick other contributors into thinking it's meant to be reused from elsewhere, which can potentially lead to messy scenarios.